### PR TITLE
Feature/gen commons pkg

### PR DESCRIPTION
**Pull request by skuhlmann**
## GitHub Issue

[GitHub issue.](https://github.com/HausDAO/daohaus-monorepo/issues/49)

## Changes

- nx gen the commons library
- directory setup: constants, legos, utils, types

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: libs/commons/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// export * from './lib/constants';
+// export * from './lib/legos';
+// export * from './lib/types';
+// export * from './lib/utils';
```
Comment (alexkeating) [inline libs/commons/src/index.ts:4]: This should be removed, probably along with the empty files

**Review by alexkeating — COMMENTED** [review]
Looks good, once the commented out code and empty files are removed this should be good to merge

File: libs/commons/src/index.ts (hunk 1 above)
Comment (skuhlmann) [reply libs/commons/src/index.ts:4]: You mind if I add one util/example file for each? I've got to bring them back right away.

Comment (skuhlmann) [reply libs/commons/src/index.ts]: Actually I just removed all the empty things. GTG!

**Review by jonathanprozzi — APPROVED** [review]
This looks good! Once we have some usage instructions we can update the Readme (same with all of our other things in Sage)

**Comment by skuhlmann** [comment]
> This looks good! Once we have some usage instructions we can update the Readme (same with all of our other things in Sage)

Yeah I'll take a stab at that in the next PR that will have actual functionality.

Maybe we need a readme template?

**Pull request merged**